pubspec.yaml

```diff
@@ -7,7 +7,7 @@ author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/browser_launcher
 
 environment:
-  sdk: '>=2.2.0 <3.0.0'
+  sdk: '>=2.1.0-dev <3.0.0'
```

Comment:
be explicit. Figure out what flutter stable is at and use that.

Update .travis using a version we published - https://www.dartlang.org/tools/sdk/archive#dev-channel